pkg/index/index.go

```diff
@@ -57,7 +61,11 @@ func New(rewriters []PathRewriter) *State {
 		// Experimental feature: allow mounts to be used with Workspaces
 		// structure: (clusterName, workspace name) -> string serialized mount objects
 		// This should be simplified once we promote this to workspace structure.
-		clusterWorkspaceMountAnnotation: map[logicalcluster.Name]map[string]string{},
+		clusterWorkspaceMountAnnotation: map[string]map[logicalcluster.Name]map[string]string{},
```

Comment:
no sure I understand the three keys. shard, cluster, workspace? Then it should be called like that.

Reply:
ah, yes,changed last second. will rename.